src/main/java/nl/ordina/jobcrawler/service/SkillService.java

```diff
@@ -94,4 +94,15 @@ public boolean delete(UUID id) {
     }
 
 
+    public void deleteReferencesToSkills() {
+        skillRepository.deleteReferencesToSkills();
+    }
+
+    public void deleteReferencesToSkill(String name) {
+        skillRepository.deleteReferencesToSkill(name);
+    }
```

Comment:
maybe it is a good idea to only have one skill and if the caller needs to delete a set, then it has to implement a loop

Reply:
no, will create extra code